src/sound/soundbase.cpp

```diff
@@ -232,98 +221,140 @@ QVector<QString> CSoundBase::LoadAndInitializeFirstValidDriver ( const bool bOpe
 }
 
 /******************************************************************************\
-* MIDI handling                                                                *
+* Command Line Handling                                                        *
 \******************************************************************************/
 void CSoundBase::ParseCommandLineArgument ( const QString& strMIDISetup )
 {
+    if ( strMIDISetup.isEmpty() )
+        return;
+
     int iMIDIOffsetFader = 70; // Behringer X-TOUCH: offset of 0x46
 
-    // parse the server info string according to definition: there is
```

Comment:
Where did this go?

Reply:
(new) Line 233.

Reply:
(`parse the server info string` was obviously copied from the parsing of the server info string - nothing to do with parsing the `ctrlmidich` string.)

Reply:
Oh, and [more documentation](https://github.com/jamulussoftware/jamuluswebsite/pull/1053) is being added.